packages/daemon/src/services/index.ts

```diff
@@ -285,6 +292,52 @@ export const handleVertexAccepted = async (context: Context, _event: Event) => {
       // update wallet_balance and wallet_tx_history tables
       const walletBalanceMap: StringMap<TokenBalanceMap> = getWalletBalanceMap(addressWalletMap, addressBalanceMap);
       await updateWalletTablesWithTx(mysql, hash, timestamp, walletBalanceMap);
+
+      const tx: Transaction = {
+        tx_id: hash,
+        nonce,
+        timestamp,
+        voided: metadata.voided_by.length > 0,
+        weight,
+        parents,
+        version,
+        inputs: inputs as unknown as TxInput[],
+        outputs: outputs as unknown as TxOutput[],
+        height: metadata.height,
+        token_name,
+        token_symbol,
+      };
+
+      try {
+        const queueUrl = NEW_TX_SQS;
+        const client = new SQSClient({});
+        const command = new SendMessageCommand({
+          QueueUrl: queueUrl,
+          MessageBody: JSON.stringify({
+            wallets: Array.from(seenWallets),
+            tx,
+          }),
+        });
+
```

Comment:
suggestion(non-blocking): Although it is simple, maybe we should extract this as a method `sendMessageSQS`?

Reply:
I agree, done in 52d1cdad79d93803d653f1cd0ce252b915635da8